hknweb/candidate/models.py

```diff
@@ -56,35 +55,25 @@ def rejected(self):
     def __str__(self):
         return self.name
 
-class CandidateForm(models.Model):
-
-    name           = models.CharField(max_length=MAX_STRLEN, default='')
-    duedate        = models.DateTimeField(default=timezone.now)
-    link           = models.CharField(max_length=MAX_STRLEN, default='')
-    # if visible == False, then admins can see candiate form but it's not displayed on portal
-    visible         = models.BooleanField(default=False)
-
-    def __str__(self):
-        return self.name
-
 
 class BitByteActivity(models.Model):
     """
-    Model for one bit byte activity for one candidate.
-    Each bite byte activity confirmation must be added separately for each
-    candidate by VP on the admin site. However, these objects can be bulk added.
+    Model for one bit byte activity for a group of candidates.
+    For each bit byte activity, one candidate must submit a request for the entire
+    group on the portal, and this needs to be confirmed by VP on the admin site.
     """
-
     class Meta:
         verbose_name_plural = "Bit Byte Activities"
 
-    candidate = models.ForeignKey('auth.User', limit_choices_to={'groups__name': settings.CAND_GROUP},
-                                on_delete=models.CASCADE, default=None)
-    notes = models.TextField(max_length=MAX_TXTLEN, blank=True, default='')
-    confirm_date = models.DateTimeField(auto_now_add=True)
+    candidates = models.ManyToManyField('auth.User')
```

Comment:
I did this on purpose in case people want to tag officers in this, it doesn't hurt. Do you think this filter is necessary?

Reply:
That makes a lot of sense! It might be confusing because in multiple places it says "Candidate usernames" such as 
![alex pr bitbyte-req bitbyteactivity admin candidate officers](https://user-images.githubusercontent.com/46059916/76124546-c1967500-5faf-11ea-981f-4b131270e42f.jpg)
![alex pr bitbyte-req bitbyteactivity candidate request_LI](https://user-images.githubusercontent.com/46059916/76124547-c22f0b80-5faf-11ea-9d9e-153564b9e521.jpg)
which may be misleading in some manner. Maybe we can specify it as "Participants" or "Members" or something?

Reply:
Hmm, it's quite a bit of work to change it all right now, probably more work than to filter out officers. But in the future it's possible we decide to track officers as well?

Reply:
For now, how does
1. Changing the wording to "Participants"
2. Opening an issue to formalize the officer tracking
sound?

Reply:
Oh no adding officer tracking is way too ambitious, that's more for like in a few semesters. I've added the "candidates" terminology in a ton of places so I'll need to do a search